### PR TITLE
Minor improvements for reference.conf

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

### CODE REVIEW

File: model/src/main/resources/reference.conf [hunk 1]
```diff
@@ -31,19 +31,19 @@ ort {
 
   licenseFilePatterns {
     licenseFilenames = ["license*"]
-    patentFilenames = ["patents"]
+    patentFilenames = [patents]
```
Comment (sschuberth) [inline model/src/main/resources/reference.conf:34]: Hmm, I'm not sure esp. about this one. Isn't it confusing to omit quotes here, but not above and below?

**Review by sschuberth — APPROVED** [review]
Approving despite my comment, as the syntax seems to be valid after all.

File: model/src/main/resources/reference.conf (hunk 1 above)
Comment (mnonnenmacher) [reply model/src/main/resources/reference.conf:34]: I first thought the same, but then decided it's more consistent to use quotes only if they are required in all places in the file. But I'd be fine with changing this one back as well if you prefer that.

Comment (sschuberth) [reply model/src/main/resources/reference.conf:34]: No, I think it's fine to keep it as a show-case that wildcard patterns need to be quoted.

**Pull request merged**